src/renderer/mod.rs

```diff
@@ -1017,8 +1019,7 @@ impl ShaderProgram {
             u_cell_dim: cell_dim,
             u_visual_bell: visual_bell,
             u_background: background,
-            padding_x: config.padding().x,
-            padding_y: config.padding().y,
+            padding: config.padding().clone(),
```

Comment:
Another clone that shouldn't be necessary, same as the other one, this can be removed by making padding `Copy`.

Reply:
For these, I've changed them to simply dereferencing `config.padding()`, since they're already `derive(Copy)`. This appears to work.